src/state-card-custom-ui.html

```diff
@@ -88,6 +128,11 @@
       window.hassUtil.applyThemesOnElement(
         this, hass.themes, modifiedObj.attributes.theme);
     }
+    if (modifiedObj.attributes.background_color) {
```

Comment:
1) You are not styling the card in `more-info` window. Is that on purpose?

2) If `attributes.background_color` becomes undefined (due to tag being reused by another entity - not sure if that can happen, or due to dynamic customization) the color will stay.
Adding an else clause will handle the dynamic customization case.

Reply:
1. Yes. At first I had it without, but that looked ugly. The reason is that the more-info includes this part, and then adds the other info below. Only the state part will be themed, not the complete more-info box, so it makes no sense. That's why I added the check to make sure the styling is only applied to the state card.

2. I see. I suppose we can add an "else" clause with a transparent background-color. Would that solve the issue?